src/SpeedJumpIrm.sol

```diff
@@ -108,7 +112,7 @@ contract AdaptativeCurveIrm is IIrm {
         return avgBorrowRate;
     }
 
-    /// @dev Returns avgBorrowRate and endRateAtTarget.
+    /// @dev Returns avgRate and endRateAtTarget.
```

Comment:
[minor] revert this change because we can now use the same name as where this function is called (in `borrowRate` and `borrowRateView`)

Reply:
should I put avgRate everywhere?

Reply:
it works too